### PR TITLE
Add docs for deployment core platform services

### PR DESCRIPTION
**Pull request by taliaga**
Orion, QL, IOTA

### CODE REVIEW

**Comment by chicco785** [comment]
hi tomas,
the point was not having pointers, but having a few line instructions to follow and say if you want more check here.

see what i provided here:
https://github.com/smartsdk/guided-tour/pull/3/files

File: docs/platform/deployservices.md [hunk 1]
```diff
@@ -1,13 +1,37 @@
 # Deploy your platform using SmartSDK recipes
 
+This section will give you pointers to deploy core services of SmartSDK using
+docker recipes, which are mantained in [this repository](https://github.com/smartsdk/smartsdk-recipes)
+ and whose documentation can be read [here](https://smartsdk.github.io/smartsdk-recipes/)
+
+If you haven't already, please take some time to read those docs so that you
+make sure you have properly setup the environment (the docker swarm cluster)
+where the deployment will be done. This is covered in [this section](deployservices.md)
```
Comment (chicco785) [inline docs/platform/deployservices.md]: is the link correct?

File: docs/platform/deployservices.md [hunk 2]
```diff
@@ -1,13 +1,37 @@
 # Deploy your platform using SmartSDK recipes
 
+This section will give you pointers to deploy core services of SmartSDK using
```
Comment (chicco785) [inline docs/platform/deployservices.md]: This section provide a quick introduction to deploy some core FIWARE services using SmartSDK recipes. In particular, we will see how you can deploy:
- xxxx a service to blah blah.
-
-
 For more details and advanced topics, check the specific documentation here.

File: docs/platform/deployservices.md (hunk 1 above)
Comment (chicco785) [reply docs/platform/deployservices.md]: Initial set-up... 

* set-up a swarm cluster as described in ....
* clone the recipe repository:
```
git clone ....
```

File: docs/platform/deployservices.md [hunk 3]
```diff
@@ -1,13 +1,37 @@
...
+
+If you haven't already, please take some time to read those docs so that you
+make sure you have properly setup the environment (the docker swarm cluster)
+where the deployment will be done. This is covered in [this section](deployservices.md)
+of this guide.
+
 ## Deploy an Highly Available Context Broker
 
-- TODO
+To deploy Orion Context Broken in HA, please refer to [this section](https://smartsdk.github.io/smartsdk-recipes/data-management/context-broker/ha/readme/) of the
```
Comment (chicco785) [inline docs/platform/deployservices.md]: To deploy the context broker.... 

```cd ....```
```nano env variable```
```sh deployxxx.sh```

wait few seconds and test:

```curl.....```

For fine grained configuration refer to...

File: docs/platform/deployservices.md [hunk 4]
```diff
@@ -1,13 +1,37 @@
...
 
 ## Deploy an Highly Available QuantumLeap
 
-- TODO
+To deploy Orion Context Broken in HA, please refer to [this section](https://smartsdk.github.io/smartsdk-recipes/data-management/quantumleap/readme/) of the
+SmartSDK recipes documentation.
+
+As with the rest of the recipes, the files mentioned in the documentation are
+those located in the same directory where the documentation file is. For this
+case, we are talking about [this directory](https://github.com/smartsdk/smartsdk-recipes/tree/master/recipes/data-management/quantumleap).
```
Comment (chicco785) [inline docs/platform/deployservices.md]: To deploy the ql.... 

```cd ....```
```nano env variable```
```sh deployxxx.sh```

wait few seconds and test:
1. create a subscpritpion in ctx
1. send data to ctx
1. query data from ql

For fine grained configuration refer to...

File: docs/platform/deployservices.md [hunk 5]
```diff
@@ -1,13 +1,37 @@
...
 
 ## Deploy an Highly Available IoT Agent
 
-- TODO
+To deploy IoT Agents in HA, please refer to [this section](https://smartsdk.github.io/smartsdk-recipes/iot-services/readme/) of the
```
Comment (chicco785) [inline docs/platform/deployservices.md]: To deploy the a iot-ul agent... (for other agents please refer to ... ) ...

```cd ....
nano env variable
sh deployxxx.sh
```
wait few seconds and test:

1. create a service in the agent
1. create device
1. create a subscpritpion in ctx
1. send data to agent
1. query data from ql
For fine grained configuration refer to...

**Review by chicco785 — CHANGES_REQUESTED** [review]

File: docs/platform/deployservices.md (hunk 1 above)
Comment (taliaga) [reply docs/platform/deployservices.md]: good catch, this was supposed to be swarmcluster.md (it's a local link)

**Comment by taliaga** [comment]
Accompanying PR: https://github.com/smartsdk/smartsdk-recipes/pull/37

File: docs/platform/deployservices.md [hunk 6]
```diff
@@ -1,13 +1,311 @@
...
+
+Crate needs a minimum of 262144 for the mmap count, which by default is lower in
+Swarm nodes. Thus, before attempting to launch this service, you should run the
+following command in each of your swarm nodes.
+
+Assuming you followed the miniswarm example, you can do...
+
+```
+$ docker-machine ssh ms-manager0
+docker@ms-manager0:~$ sudo sysctl -w vm.max_map_count=262144
```
Comment (taliaga) [inline docs/platform/deployservices.md:195]: @dancn is this something we should take care in the platform docs maybe?

Comment (dancn) [reply docs/platform/deployservices.md:195]: @taliaga I will take care of adding this in the platform docs and to add this line in our custom deployer script.

Comment (taliaga) [reply docs/platform/deployservices.md:195]: ping me when you have news so I recall to remove it from here afterwards :)

**Review by chicco785 — APPROVED** [review]

**Pull request merged**